BUILD.md

```diff
@@ -135,6 +133,12 @@ cd cactus
 npm run enable-corepack
 ```
 
+* Run this command to migrate lockfile (Also, run this everytime you have changes before running `yarn run cofugure`)
```

Comment:
@zondervancalvez I'm guessing that's a typo in the script name, please fix!

Reply:
After some changes and re-testing, we already removed this.

Reply:
@zondervancalvez It is still in the diff.